composer.json

```diff
@@ -24,6 +24,9 @@
         "twig/twig": "~1.28|~2.0",
         "psr/log": "~1.0"
     },
+    "conflict": {
+        "phpunit/phpunit": "< 4.8.35 || >=5.0 <=5.4.2"
```

Comment:
Symfony 2.7 supports php 5.3
This rule prevent people to install a version of phpunit compatible with php < 5.6